src/landscape/components/LandscapeForm/AffiliationStep.js

```diff
@@ -45,6 +47,9 @@ const FORM_FIELDS = [
   {
     name: 'partnership.year',
     label: 'landscape.form_profile_partnership_year',
+    helperText: {
+      i18nKey: 'landscape.form_profile_partnership_year_helper_text',
+    },
```

Comment:
question: do you anticipate additional keys inside `helperText`?

Reply:
@paulschreiber More than additional keys the helper text component has more parameters that can be used to cutomize how it works